lib/jazzy/podspec_documenter.rb

```diff
@@ -0,0 +1,109 @@
+require 'tmpdir'
+require 'json'
+
+module Jazzy
+  class PodspecDocumenter
+    attr_reader :podspec
+
+    def initialize(podspec)
+      @podspec = podspec
+    end
+
+    def sourcekitten_output
+      sandbox = Pod::Sandbox.new(pod_config.sandbox_root)
+      installer = Pod::Installer.new(sandbox, podfile)
+      installer.install!
+      stdout = Dir.chdir(sandbox.root) do
+        pod_targets.map do |t|
+          SourceKitten.run_sourcekitten(
+            %W(doc --module-name #{podspec.module_name} -target #{t}),
+          )
+        end
+      end
+      stdout.reduce([]) { |a, s| a + JSON.load(s) }.to_json
+    end
+
+    def self.configure(config, podspec)
+      case podspec
+      when Pathname, String
+        require 'cocoapods'
+        podspec = Pod::Specification.from_file(podspec)
+      end
+
+      return unless podspec
+
+      config.author_name = author_name(podspec)
+      config.module_name = podspec.module_name
+      config.author_url = podspec.homepage || ''
```

Comment:
Won't this create empty links if `homepage` isn't set? Seems like there shouldn't be a default here.